apis/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -24,7 +24,8 @@ const (
 	defaultLogTempStoragePath            string = "/var/lib/datadog-agent/logs"
 
 	// defaultLiveProcessCollectionEnabled   bool = false
-	defaultLiveContainerCollectionEnabled bool = true
+	defaultLiveProcessCollectionScrubArguments bool = true
+	defaultLiveContainerCollectionEnabled      bool = true
```

Comment:
This contradicts what we have in `docs/configuration.v2alpha1.md`:
`| features.liveProcessCollection.enabled | Enabled enables Process monitoring. Default: false`